recipes/newrelic/infrastructure/logs/ubuntu-logs.yml

```diff
@@ -0,0 +1,74 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: logs-integration
+displayName: Logs integration
+description: New Relic install recipe for basic Logging via Infra-Agent
+repository: https://github.com/newrelic/infrastructure-agent/tree/master/assets/examples/logging
+
+installTargets:
+  - type: host
+    os: linux
```

Comment:
I think this should be more specific to `debian`/`ubuntu` platforms

Reply:
Ho this is a bug, thanks for catching it, I've just added the platform